crates/vfs-notify/src/lib.rs

```diff
@@ -83,9 +83,10 @@ impl NotifyActor {
                         self.watcher = None;
                         if !config.watch.is_empty() {
                             let (watcher_sender, watcher_receiver) = unbounded();
-                            let watcher = log_notify_error(RecommendedWatcher::new(move |event| {
-                                watcher_sender.send(event).unwrap()
-                            }));
+                            let watcher =
```

Comment:
This used to say `Watcher::new_immediate`, TBH I'm not even sure how that works :sweat_smile:.